scriptlets/install_checkbox_snaps

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Installs Checkbox snap (runtime + frontend) on the DUT. Also installs
+# Checkbox on the agent from source matching the version on the DUT
+
+if [[ "$#" != "4" ]]; then
+  echo "Usage: $(basename ${BASH_SOURCE[0]}) risk runtime_name frontend_name frontend_track"
+  exit 1
+fi
+
+echo "Installing Checkbox snaps"
+
+RISK=$1
+shift
+RUNTIME_NAME=$1
+shift
+FRONTEND_NAME=$1
+shift
+FRONTEND_TRACK=$1
+shift
```

Comment:
How about removing these `shift`s and accessing `$1` to `$4` directly?

Reply:
makes sense, idk why I did it this way